### PR TITLE
14.0 mig stock financial risk

### PR DESCRIPTION
**Pull request by psugne**
https://hive.versada.eu/projects/baltled/work_packages/13605

stock _financial_risk migration to v14.0

Please check [a990b59](https://github.com/versada/credit-control/commit/a990b59fe3fd8346aec62b580efdaefb09ff5c86) commit

### CODE REVIEW

File: stock_financial_risk/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            moves = self.filtered(
+                lambda x: (
+                    x.location_dest_id.usage == "customer"
+                    and x.partner_id.commercial_partner_id.risk_exception
+                )
+            )
+            if moves:
+                raise exceptions.UserError(
+                    _("Financial risk exceeded in partner:\n%s")
+                    % moves.mapped("partner_id.name")
```
Comment (naglis) [inline stock_financial_risk/models/stock.py]: Nitpick: In Odoo 14.0, `_()` supports passing arguments to it:
```suggestion
                    _(
                        "Financial risk exceeded in partner:\n%s",
                        moves.mapped("partner_id.name"),
                    )
```

File: stock_financial_risk/models/stock.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                )
+        return super(StockMove, self)._action_done()
```
Comment (naglis) [inline stock_financial_risk/models/stock.py]: Nitpick: can skip passing arguments to `super` (same for `StockPicking` methods)
```suggestion
        return super()._action_done()
```

File: stock_financial_risk/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Dauden
+  * Pedro M. Baeza
+  * David Vidal
+  * Ernesto Tejeda
+
+* `Jarsa <https://www.jarsa.com.mx>`_:
+
+  * Alan Ramos
```
Comment (naglis) [inline stock_financial_risk/readme/CONTRIBUTORS.rst:10]: If this will be contributed back to OCA, perhaps you ought to include yourself as contributor?

**Review by naglis — APPROVED** [review]

File: stock_financial_risk/models/stock.py (hunk 1 above)
Comment (psugne) [reply stock_financial_risk/models/stock.py]: TIL :nerd_face:

**Review by paulius-sladkevicius — APPROVED** [review]

**Pull request merged**